packages/@aws-cdk/aws-opensearchservice/test/integ.opensearch.imported-vpc.ts

```diff
@@ -0,0 +1,65 @@
+/// !cdk-integ * pragma:enable-lookups
```

Comment:
Please update the integ test to use the lasted style (I know, it's a tad confusing).

You'll need to drop this line and at the end needs something like this
```ts
new integ.IntegTest(app, '...', {
  testCases: [stack],
});
```

See: https://github.com/aws/aws-cdk/blob/main/INTEGRATION_TESTS.md